README.txt
Have a look at the manual in the doc folder.

Extension icon is under Public Domain:
http://openiconlibrary.sourceforge.net/gallery2/?./Icons/mimetypes/video-x-generic.png

Source: tango , License: PD 

ChangeLog
2013-09-14 Hauke Hain  <hhpreuss@googlemail.com>

	* Added compatibility for TYPO3 4.5.x to 6.2.x.
		Does not support DAM in TYPO3 6+.

2010-11-14 Hauke Hain  <hhpreuss@googlemail.com>

	* Multiple hpfavideo plugins on the same page are now possible

2010-11-13 Hauke Hain  <hhpreuss@googlemail.com>

	* first release

2010-11-12 Hauke Hain  <hhpreuss@googlemail.com>

	* initial code generated with kickstarter

ext_tables.php
<?php
if (!defined ('TYPO3_MODE')) {
	die ('Access denied.');
}


$version=class_exists('t3lib_utility_VersionNumber')
	? t3lib_utility_VersionNumber::convertVersionNumberToInteger(TYPO3_version)
	: t3lib_div::int_from_ver(TYPO3_version);
if ($version < 6001000) {
	t3lib_div::loadTCA('tt_content');
}

$TCA['tt_content']['types']['list']['subtypes_excludelist'][$_EXTKEY.'_pi1']='layout,select_key,pages';


t3lib_extMgm::addPlugin(array(
	'LLL:EXT:hpfavideo/locallang_db.xml:tt_content.list_type_pi1',
	$_EXTKEY . '_pi1',
	t3lib_extMgm::extRelPath($_EXTKEY) . 'ext_icon.gif'
),'list_type');


if (TYPO3_MODE == 'BE') {
	$TBE_MODULES_EXT['xMOD_db_new_content_el']['addElClasses']['tx_hpfavideo_pi1_wizicon'] = t3lib_extMgm::extPath($_EXTKEY).'pi1/class.tx_hpfavideo_pi1_wizicon.php';
}

$TCA['tt_content']['types']['list']['subtypes_addlist'][$_EXTKEY.'_pi1']='pi_flexform';
t3lib_extMgm::addPiFlexFormValue($_EXTKEY.'_pi1', 'FILE:EXT:'. $_EXTKEY. '/pi1/flexform_ds.xml');

t3lib_extMgm::addStaticFile($_EXTKEY, 'static/simplePlaylist', 'hpfavideo: Simple Playlist');

?>

pi1/class.tx_hpfavideo_pi1.php
<?php
/***************************************************************
*  Copyright notice
*
*  (c) 2010-2013 Hauke Hain <hhpreuss@googlemail.com>
*  All rights reserved
*
*  This script is part of the TYPO3 project. The TYPO3 project is
*  free software; you can redistribute it and/or modify
*  it under the terms of the GNU General Public License as published by
*  the Free Software Foundation; either version 2 of the License, or
*  (at your option) any later version.
*
*  The GNU General Public License can be found at
*  http://www.gnu.org/copyleft/gpl.html.
*
*  This script is distributed in the hope that it will be useful,
*  but WITHOUT ANY WARRANTY; without even the implied warranty of
*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
*  GNU General Public License for more details.
*
*  This copyright notice MUST APPEAR in all copies of the script!
***************************************************************/
/**
 * [CLASS/FUNCTION INDEX of SCRIPT]
 *
 *
 *
 *   55: class tx_hpfavideo_pi1 extends tslib_pibase
 *   69:     function main($content, $conf)
 *  154:     private function showPlayer()
 *  199:     private function getPlayerInitialisationJSCode()
 *  323:     private function getFirstVideo()
 *  338:     private function getFirstVideoPreviewImage()
 *  353:     private function getPlaylistJavaScriptFunctions()
 *  372:     private function getPlaylist()
 *  406:     private function getVideosInformationArray()
 *  435:     private function getSingle($data, $ts)
 *
 * TOTAL FUNCTIONS: 9
 * (This index is automatically created/updated by the extension "extdeveval")
 *
 */

/**
 * Plugin 'FAVideo player' for the 'hpfavideo' extension.
 *
 * @author	Hauke Hain <hhpreuss@googlemail.com>
 * @package	TYPO3
 * @subpackage	tx_hpfavideo
 */
class tx_hpfavideo_pi1 extends tslib_pibase {
	var $prefixId		  = 'tx_hpfavideo_pi1';		// Same as class name
	var $scriptRelPath = 'pi1/class.tx_hpfavideo_pi1.php';	// Path to this script relative to the extension dir.
	var $extKey				= 'hpfavideo';	// The extension key.
	var $pi_checkCHash = true;
	var $uploadFolder  = 'uploads/tx_hpfavideo/';

	/**
	 * The main method of the PlugIn
	 *
	 * @param	string		$content: The PlugIn content
	 * @param	array		$conf: The PlugIn configuration
	 * @return	string		The content that is displayed on the website
	 */
	function main($content, $conf) {
		$this->conf = $conf;
		$this->pi_setPiVarDefaults();
		$this->pi_loadLL();

		$this->siteRelPath = t3lib_extMgm::siteRelPath($this->extKey);
		$this->playerID = 'playerID' . $this->cObj->data['uid'];

		// load TypoScript values
		$this->basePath = $this->conf['basePath'];

		// if no valid URL is set by TypoScript use server value
		if (!t3lib_div::isValidUrl($this->basePath)) {
			$this->basePath = 'http://'.$GLOBALS['_SERVER']['HTTP_HOST'];
		}

		// check wheather DAM is loaded
		$this->DAMloaded = t3lib_extMgm::isLoaded('dam');

		// load flexform values
		$this->pi_initPIflexForm();

		// sheet video
		$this->flvfile = $this->pi_getFFvalue($this->cObj->data['pi_flexform'], 'flvfile', 'video');
		$this->previmg = $this->pi_getFFvalue($this->cObj->data['pi_flexform'], 'previmg', 'video');
		$this->playerwidth = (int) $this->pi_getFFvalue($this->cObj->data['pi_flexform'], 'playerwidth', 'video');
		$this->playerheight = (int) $this->pi_getFFvalue($this->cObj->data['pi_flexform'], 'playerheight', 'video');
		$this->videoScaleMode = $this->pi_getFFvalue($this->cObj->data['pi_flexform'], 'videoScaleMode', 'video');

		if ($this->DAMloaded) {
			$this->flvfiledam = tx_dam_db::getReferencedFiles('tt_content', $this->cObj->data['uid'], 'hpfavideo_flv', 'tx_dam_mm_ref');
			$this->previmgdam = tx_dam_db::getReferencedFiles('tt_content', $this->cObj->data['uid'], 'hpfavideo_img', 'tx_dam_mm_ref');
		}

		// sheet display
		$this->skinPath = $this->pi_getFFvalue($this->cObj->data['pi_flexform'], 'skin', 'display');
		$this->controlColor = '0x' . str_replace('#', '', $this->pi_getFFvalue($this->cObj->data['pi_flexform'], 'controlColor', 'display'));
		$this->backgroundColor = str_replace('#', '', $this->pi_getFFvalue($this->cObj->data['pi_flexform'], 'backgroundColor', 'display'));
		$this->videoAlign = $this->pi_getFFvalue($this->cObj->data['pi_flexform'], 'videoAlign', 'display');
		$this->skinAutoHide = $this->pi_getFFvalue($this->cObj->data['pi_flexform'], 'skinAutoHide', 'display');
		$this->skinVisible = $this->pi_getFFvalue($this->cObj->data['pi_flexform'], 'skinVisible', 'display');

		// sheet options
		$this->clickToTogglePlay = $this->pi_getFFvalue($this->cObj->data['pi_flexform'], 'clickToTogglePlay', 'options');
		$this->autoPlay = $this->pi_getFFvalue($this->cObj->data['pi_flexform'], 'autoPlay', 'options');
		$this->autoLoad = $this->pi_getFFvalue($this->cObj->data['pi_flexform'], 'autoLoad', 'options');
		$this->volume = (int) $this->pi_getFFvalue($this->cObj->data['pi_flexform'], 'volume', 'options');

 		// overwrite properties
 		if ($this->videoScaleMode == 'default') {
			 $this->videoScaleMode = $this->conf['videoScaleMode'];
		}

 		if ($this->videoAlign == 'default') {
			 $this->videoAlign = $this->conf['videoAlign'];
		}

 		if ($this->skinPath == 'default') {
			 $this->skinPath = $this->conf['skinPath'];
		} else {
			$this->skinPath = 	$this->basePath. '/' . $this->siteRelPath .
								'res/skins/' . $this->skinPath;
		}

 		if ($this->volume == 0) {
			 $this->volume = (int) $this->conf['volume'];
		}

 		if (strlen($this->controlColor) < 8) {
			 $this->controlColor = $this->conf['controlColor'];
		}

 		if (strlen($this->backgroundColor) < 6) {
			 $this->backgroundColor = $this->conf['backgroundColor'];
		}

		//return $this->pi_wrapInBaseClass($this->showPlayer());
		return $this->showPlayer();
	}

	/**
	 * Includes all needed JavaScript and created needed HTML.
	 *
	 * @return	string		HTML
	 */
	private function showPlayer() {
		// Print error if no flv file is set
		if (strlen($this->flvfile) < 1 && strlen(current($this->flvfiledam['files'])) < 1) {
			$content = $this->cObj->stdWrap($this->pi_getLL('noFlvFile'), $this->conf['errorWrap.']);
		} else {
			// include JS files
			$GLOBALS['TSFE']->pSetup['includeJSFooter.'][$this->extKey . '_favideo']  = $this->siteRelPath . 'res/FAVideo.js';
			$GLOBALS['TSFE']->pSetup['includeJSFooter.'][$this->extKey . '_AC_RunActiveContent']  = $this->siteRelPath . 'res/AC_RunActiveContent.js';

			// include JavaScript for initialisating the player
			$GLOBALS['TSFE']->pSetup['jsFooterInline.']['815'] = 'TEXT';
			$GLOBALS['TSFE']->pSetup['jsFooterInline.']['815.']['value'] .= $this->getPlayerInitialisationJSCode();

			// Div for player insertion with error message in case JavaScript is not enabled
			$player =	'<div id="' . $this->playerID . 'Ident">' .
							$this->cObj->stdWrap($this->pi_getLL('noJavaScript'), $this->conf['errorWrap.']) .
						'</div>';

			// show playlist
			if ($this->DAMloaded) {
				$playlist = $this->getPlaylist();
				$GLOBALS['TSFE']->pSetup['jsFooterInline.']['815.']['value'] .= $this->getPlaylistJavaScriptFunctions();
			}

			// if Typoscript alignment  is set use it
			if (isset($this->conf['alignment']) && isset($this->conf['alignment.'])) {
				$content .= $this->getSingle(
							array(
								'player'	=> $player,
								'playlist'	=> $playlist
								),
							'alignment');
			} else { // otherwise output the player and then the playlist
				$content .= $player . $playlist;
			}
		}

		return $content;
	}

	/**
	 * Initialises the player via JavaScript with all options.
	 *
	 * @return	string		JavaScript to initialise the player
	 */
	private function getPlayerInitialisationJSCode() {
		// These options are currently not used:
		// playHeadTime, totalTime, bufferTime, playheadUpdateInterval
		$options = 'autoLoad:';

		if ($this->autoLoad) {
			$options .= 'true';
		} else {
			$options .= 'false';
		}

		$options .= ', autoPlay:';

		if ($this->autoPlay) {
			$options .= 'true';
		} else {
			$options .= 'false';
		}

		if ($this->volume > 0) {
			$options .= ', volume:' . $this->volume;
		}

		$options .= ', skinAutoHide:';

		if ($this->skinAutoHide) {
			$options .= 'true';
		} else {
			$options .= 'false';
		}

		$options .= ', skinVisible:';

		if ($this->skinVisible) {
			$options .= 'true';
		} else {
			$options .= 'false';
		}

		$options .= ', clickToTogglePlay:';

		if ($this->clickToTogglePlay) {
			$options .= 'true';
		} else {
			$options .= 'false';
		}

		if (strlen($this->skinPath) > 10) {
			$options .= ', skinPath:\'' . $this->skinPath . '\'';
		}

		switch ($this->videoScaleMode) {
			case 'maintainAspectRatio':
			case 'exactFit':
			case 'noScale':
				$options .= ', videoScaleMode:\'' . $this->videoScaleMode . '\'';
				break;
		}

		switch ($this->videoAlign) {
			case 'bottom':
			case 'bottomLeft':
			case 'bottomRight':
			case 'top':
			case 'topLeft':
			case 'topRight':
			case 'center':
			case 'right':
			case 'left':
				$options .= ', videoAlign:\'' . $this->videoAlign . '\'';
				break;
		}

		$options .= ', previewImagePath:\'' .
					$this->getFirstVideoPreviewImage() . '\'';

		if (strlen($this->backgroundColor) <> 6) {
			$this->backgroundColor = '000000';
		}

		$js =	$this->playerID .
				' = new FAVideo(\'' . $this->playerID . 'Ident\', ' .
				'\'' . $this->getFirstVideo() . '\', ' . $this->playerwidth .
				', ' . $this->playerheight . ' , { ' . $options . ' }, \'' .
				$this->basePath. '/' . $this->siteRelPath . 'res/\', \'' .
				$this->backgroundColor . '\');';

		if (strlen($this->controlColor) == 8) {
			$js .=  $this->playerID .
					'.setThemeColor(\'' . $this->controlColor . '\');';
		}

		// Loading problems (not ready DOM):
		// Register Listeners and unregister it after it is not needed anymore
		// if the player plays the video it gets scaled (again)
		$js .=	$this->playerID . '.addEventListener(\'stateChange\', this, ' .
				$this->playerID . 'stateChange);' .
					'function ' . $this->playerID . 'stateChange(event) {' .
						'if (event.state ==  \'playing\') {' . $this->playerID .
							'.setVideoScaleMode(' . $this->playerID .
								'.getVideoScaleMode());' .
							$this->playerID .
							'.removeEventListener(\'stateChange\', this, ' .
							$this->playerID . 'stateChange);' .
						'}' .
					'}';
		// after the initialisation the color theme gets applied again and the
		$js .=	$this->playerID .'.addEventListener(\'init\', this, ' .
				$this->playerID . 'init);' .
					'function ' . $this->playerID . 'init(event) {' .
							$this->playerID . '.setThemeColor(' .
								$this->playerID . '.getThemeColor());' .
						$this->playerID .'.removeEventListener(\'init\', this, ' .
						$this->playerID . 'init);' .
					'}';

		return $js;
	}

	/**
	 * Returns the path of path to the first video
	 *
	 * @return	string		path to the first video
	 */
	private function getFirstVideo() {
		$filePath = $this->uploadFolder . $this->flvfile;

		if ($this->DAMloaded) {
			$filePath = current($this->flvfiledam['files']);
		}

		return $this->basePath. '/' . $filePath;
	}

	/**
	 * Returns the path of path to the first preview image.
	 *
	 * @return	string		path to the first preview image
	 */
	private function getFirstVideoPreviewImage() {
		$filePath = $this->uploadFolder . $this->previmg;

		if ($this->DAMloaded) {
			$filePath = current($this->previmgdam['files']);
		}

		return $this->basePath. '/' . $filePath;
	}

	/**
	 * Returns JavaScript functions needed by the playlist.
	 *
	 * @return	string		JavaScript functions
	 */
	private function getPlaylistJavaScriptFunctions() {
		return 	'function loadJAVideo' . $this->playerID . '(videourl, imgurl) {' .
					$this->playerID . '.setPreviewImagePath(imgurl);' .
					$this->playerID .'.load(videourl);' .
					$this->playerID .
					'.addEventListener(\'stateChange\', this, ' .
					$this->playerID . 'stateChange);' .
				'}' .
				'function playJAVideo' . $this->playerID . '(videourl, imgurl) {' .
					'loadJAVideo' . $this->playerID . '(videourl, imgurl);' .
					$this->playerID . '.play(videourl);' .
				'}';
	}

	/**
	 * Returns the TypoScript created playlist
	 *
	 * @return	string		HTML playlist created by TypoScript
	 */
	private function getPlaylist() {
		$videos = $this->getVideosInformationArray();
		$playlist = '';

		// create a playlist only if there are at least 2 videos
		if (sizeof($videos) > 1) {
			if (isset($this->conf['playlist']) && isset($this->conf['playlist.']) && isset($this->conf['playlistItem']) && isset($this->conf['playlistItem.'])) {
				$playlistElements = '';

				foreach ($videos as $video) {
					$playlistElements .= $this->getSingle($video, 'playlistItem');
				}

				$playlist = $this->getSingle(
							array(
								'playerID'		=> $this->playerID,
								'title'			=> $this->pi_getLL('playlistTitle'),
								'description'	=> $this->pi_getLL('playlistDescription'),
								'playlistItems'	=> $playlistElements
								),
							'playlist');
			} else {
				$playlist = $this->cObj->stdWrap($this->pi_getLL('noTypoScriptSet'), $this->conf['errorWrap.']);
			}
		}

		return $playlist;
	}

	/**
	 * Combines the DAM arrays for the image und video and returns only the needed information.
	 *
	 * @return	array		Array with the title and path of the video, path of the preview image and the DAm uid of both.
	 */
	private function getVideosInformationArray() {
		$videos = array();
		$i = 0;

		foreach ($this->flvfiledam['rows'] as $uid => $video) {
			$videos[$i++] = array(
							'playerID'	=> $this->playerID,
							'title' 	=> $video['title'],
							'videopath' => $this->basePath. '/' . $this->flvfiledam['files'][$uid],
							'videouid' => $uid);
		}

		$i = 0;

		foreach ($this->previmgdam['rows'] as $uid => $bild) {
			$videos[$i]['imageuid'] = $uid;
			$videos[$i++]['imagepath'] = $this->basePath. '/' . $this->previmgdam['files'][$uid];
		}

		return $videos;
	}

	/**
	 * Parses data through typoscript.
	 *
	 * @param	array		$data: Data which will be passed to the typoscript.
	 * @param	string		$ts: The typoscript which will be called.
	 * @return	[type]		...
	 */
	private function getSingle($data, $ts) {
		$cObj = t3lib_div::makeInstance('tslib_cObj');

		//Set the data array in the local cObj. This data will be available in the ts. E.G. {field:[fieldName]} or field = [fieldName]
		$cObj->data = $data;

		//Parse and return the result.
		return $cObj->cObjGetSingle($this->conf[$ts], $this->conf[$ts.'.']);
	}
}


if (defined('TYPO3_MODE') && $TYPO3_CONF_VARS[TYPO3_MODE]['XCLASS']['ext/hpfavideo/pi1/class.tx_hpfavideo_pi1.php'])	{
	include_once($TYPO3_CONF_VARS[TYPO3_MODE]['XCLASS']['ext/hpfavideo/pi1/class.tx_hpfavideo_pi1.php']);
}

?>

pi1/class.tx_hpfavideo_pi1_wizicon.php
<?php
/***************************************************************
*  Copyright notice
*
*  (c) 2010-2013 Hauke Hain <hhpreuss@googlemail.com>
*  All rights reserved
*
*  This script is part of the TYPO3 project. The TYPO3 project is
*  free software; you can redistribute it and/or modify
*  it under the terms of the GNU General Public License as published by
*  the Free Software Foundation; either version 2 of the License, or
*  (at your option) any later version.
*
*  The GNU General Public License can be found at
*  http://www.gnu.org/copyleft/gpl.html.
*
*  This script is distributed in the hope that it will be useful,
*  but WITHOUT ANY WARRANTY; without even the implied warranty of
*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
*  GNU General Public License for more details.
*
*  This copyright notice MUST APPEAR in all copies of the script!
***************************************************************/
/**
 * [CLASS/FUNCTION INDEX of SCRIPT]
 *
 * Hint: use extdeveval to insert/update function index above.
 */




/**
 * Class that adds the wizard icon.
 *
 * @author	Hauke Hain <hhpreuss@googlemail.com>
 * @package	TYPO3
 * @subpackage	tx_hpfavideo
 */
class tx_hpfavideo_pi1_wizicon {

					/**
					 * Processing the wizard items array
					 *
					 * @param	array		$wizardItems: The wizard items
					 * @return	Modified array with wizard items
					 */
					function proc($wizardItems)	{
						global $LANG;

						$LL = $this->includeLocalLang();

						$wizardItems['plugins_tx_hpfavideo_pi1'] = array(
							'icon'=>t3lib_extMgm::extRelPath('hpfavideo').'pi1/ce_wiz.gif',
							'title'=>$LANG->getLLL('pi1_title',$LL),
							'description'=>$LANG->getLLL('pi1_plus_wiz_description',$LL),
							'params'=>'&defVals[tt_content][CType]=list&defVals[tt_content][list_type]=hpfavideo_pi1'
						);

						return $wizardItems;
					}

					/**
					 * Reads the [extDir]/locallang.xml and returns the $LOCAL_LANG array found in that file.
					 *
					 * @return	The array with language labels
					 */
					function includeLocalLang()	{
						$llFile = t3lib_extMgm::extPath('hpfavideo').'locallang.xml';
						$version=class_exists('t3lib_utility_VersionNumber')
							? t3lib_utility_VersionNumber::convertVersionNumberToInteger(TYPO3_version)
							: t3lib_div::int_from_ver(TYPO3_version);
						if ($version >= 4007000) {
							$localizationParser = t3lib_div::makeInstance('t3lib_l10n_parser_Llxml');
							$LOCAL_LANG = $localizationParser->getParsedData($llFile, $GLOBALS['LANG']->lang);
						} else {
							$LOCAL_LANG = t3lib_div::readLLXMLfile($llFile, $GLOBALS['LANG']->lang);
						}
						return $LOCAL_LANG;
					}
				}



if (defined('TYPO3_MODE') && $TYPO3_CONF_VARS[TYPO3_MODE]['XCLASS']['ext/hpfavideo/pi1/class.tx_hpfavideo_pi1_wizicon.php'])	{
	include_once($TYPO3_CONF_VARS[TYPO3_MODE]['XCLASS']['ext/hpfavideo/pi1/class.tx_hpfavideo_pi1_wizicon.php']);
}

?>

ext_emconf.php
<?php

########################################################################
# Extension Manager/Repository config file for ext "hpfavideo".
#
# Auto generated 14-11-2010 18:33
#
# Manual updates:
# Only the data in the array - everything else is removed by next
# writing. "version" and "dependencies" must not be touched!
########################################################################

$EM_CONF[$_EXTKEY] = array(
	'title' => 'Flash Ajax driven videoplayer (FAVideo)',
	'description' => 'Easy to use implementation of the Flash JavaScript player "FAVideo".
More information: http://opensource.adobe.com/wiki/display/favideo/Flash-Ajax+Video+Component',
	'category' => 'plugin',
	'author' => 'Hauke Hain',
	'author_email' => 'hhpreuss@googlemail.com',
	'shy' => '',
	'dependencies' => 'cms',
	'conflicts' => '',
	'priority' => '',
	'module' => '',
	'state' => 'stable',
	'internal' => '',
	'uploadfolder' => 0,
	'createDirs' => 'uploads/tx_hpfavideo/',
	'modify_tables' => '',
	'clearCacheOnLoad' => 0,
	'lockType' => '',
	'author_company' => '',
	'version' => '1.1.2',
	'constraints' => array(
		'depends' => array(
			'cms' => '',
			'typo3' => '4.4-6.2.99',
		),
		'conflicts' => array(
		),
		'suggests' => array(
			'dam' => '1.1.5',
		),
	),
	'_md5_values_when_last_written' => 'a:77:{s:9:"ChangeLog";s:4:"55ac";s:10:"README.txt";s:4:"6e6c";s:12:"ext_icon.gif";s:4:"637b";s:17:"ext_localconf.php";s:4:"86e6";s:14:"ext_tables.php";s:4:"4987";s:13:"locallang.xml";s:4:"acbc";s:16:"locallang_db.xml";s:4:"2c4a";s:14:"doc/manual.sxw";s:4:"b2f5";s:14:"pi1/ce_wiz.gif";s:4:"d2c2";s:30:"pi1/class.tx_hpfavideo_pi1.php";s:4:"3718";s:38:"pi1/class.tx_hpfavideo_pi1_wizicon.php";s:4:"7429";s:13:"pi1/clear.gif";s:4:"cc11";s:19:"pi1/flexform_ds.xml";s:4:"6872";s:17:"pi1/locallang.xml";s:4:"8308";s:26:"res/AC_RunActiveContent.js";s:4:"ddbe";s:15:"res/FAVideo.fla";s:4:"6be3";s:14:"res/FAVideo.js";s:4:"a680";s:15:"res/FAVideo.swf";s:4:"7b28";s:40:"res/classes/com/adobe/favideo/FAVideo.as";s:4:"2480";s:43:"res/classes/com/adobe/favideo/VideoAlign.as";s:4:"cd24";s:47:"res/classes/com/adobe/favideo/VideoScaleMode.as";s:4:"66e6";s:43:"res/classes/com/adobe/favideo/VideoState.as";s:4:"28d2";s:51:"res/classes/com/adobe/favideo/managers/UIManager.as";s:4:"dacb";s:50:"res/classes/com/adobe/favideo/utils/IdleWatcher.as";s:4:"51cd";s:43:"res/classes/com/adobe/favideo/utils/Tick.as";s:4:"03d9";s:47:"res/classes/com/adobe/favideo/views/BaseView.as";s:4:"77db";s:44:"res/classes/com/adobe/favideo/views/Image.as";s:4:"385c";s:31:"res/skins/ArcticExternalAll.swf";s:4:"729e";s:33:"res/skins/ArcticExternalNoVol.swf";s:4:"c06e";s:36:"res/skins/ArcticExternalPlayMute.swf";s:4:"e687";s:40:"res/skins/ArcticExternalPlaySeekMute.swf";s:4:"08ab";s:27:"res/skins/ArcticOverAll.swf";s:4:"d198";s:29:"res/skins/ArcticOverNoVol.swf";s:4:"6511";s:32:"res/skins/ArcticOverPlayMute.swf";s:4:"9a31";s:36:"res/skins/ArcticOverPlaySeekMute.swf";s:4:"ad60";s:30:"res/skins/ClearExternalAll.swf";s:4:"0e3e";s:32:"res/skins/ClearExternalNoVol.swf";s:4:"a196";s:35:"res/skins/ClearExternalPlayMute.swf";s:4:"cf4c";s:39:"res/skins/ClearExternalPlaySeekMute.swf";s:4:"fe19";s:26:"res/skins/ClearOverAll.swf";s:4:"c5f0";s:28:"res/skins/ClearOverNoVol.swf";s:4:"d4a6";s:31:"res/skins/ClearOverPlayMute.swf";s:4:"509a";s:35:"res/skins/ClearOverPlaySeekMute.swf";s:4:"19ea";s:31:"res/skins/MojaveExternalAll.swf";s:4:"f9c6";s:33:"res/skins/MojaveExternalNoVol.swf";s:4:"84a9";s:36:"res/skins/MojaveExternalPlayMute.swf";s:4:"d27f";s:40:"res/skins/MojaveExternalPlaySeekMute.swf";s:4:"5c16";s:27:"res/skins/MojaveOverAll.swf";s:4:"0759";s:29:"res/skins/MojaveOverNoVol.swf";s:4:"3918";s:32:"res/skins/MojaveOverPlayMute.swf";s:4:"d097";s:36:"res/skins/MojaveOverPlaySeekMute.swf";s:4:"3600";s:30:"res/skins/SteelExternalAll.swf";s:4:"5f52";s:32:"res/skins/SteelExternalNoVol.swf";s:4:"b2b2";s:35:"res/skins/SteelExternalPlayMute.swf";s:4:"a090";s:39:"res/skins/SteelExternalPlaySeekMute.swf";s:4:"eab5";s:26:"res/skins/SteelOverAll.swf";s:4:"f45d";s:28:"res/skins/SteelOverNoVol.swf";s:4:"8dc3";s:31:"res/skins/SteelOverPlayMute.swf";s:4:"7c43";s:35:"res/skins/SteelOverPlaySeekMute.swf";s:4:"b1cf";s:49:"res/skins/javascript/simpleSkin/controlBar_bg.gif";s:4:"45d8";s:60:"res/skins/javascript/simpleSkin/controlBar_nextButtonOff.gif";s:4:"4bc1";s:59:"res/skins/javascript/simpleSkin/controlBar_nextButtonOn.gif";s:4:"219b";s:61:"res/skins/javascript/simpleSkin/controlBar_pauseButtonOff.gif";s:4:"e872";s:60:"res/skins/javascript/simpleSkin/controlBar_pauseButtonOn.gif";s:4:"232f";s:60:"res/skins/javascript/simpleSkin/controlBar_playButtonOff.gif";s:4:"712b";s:59:"res/skins/javascript/simpleSkin/controlBar_playButtonOn.gif";s:4:"45e1";s:60:"res/skins/javascript/simpleSkin/controlBar_prevButtonOff.gif";s:4:"520c";s:59:"res/skins/javascript/simpleSkin/controlBar_prevButtonOn.gif";s:4:"dc80";s:56:"res/skins/javascript/simpleSkin/controlBar_roundLeft.gif";s:4:"00cb";s:57:"res/skins/javascript/simpleSkin/controlBar_roundRight.gif";s:4:"cc28";s:60:"res/skins/javascript/simpleSkin/controlBar_stopButtonOff.gif";s:4:"f2e3";s:59:"res/skins/javascript/simpleSkin/controlBar_stopButtonOn.gif";s:4:"6442";s:62:"res/skins/javascript/simpleSkin/controlBar_volumeButtonOff.gif";s:4:"0cf0";s:61:"res/skins/javascript/simpleSkin/controlBar_volumeButtonOn.gif";s:4:"3332";s:61:"res/skins/javascript/simpleSkin/controlBar_volumeMutedOff.gif";s:4:"33ec";s:60:"res/skins/javascript/simpleSkin/controlBar_volumeMutedOn.gif";s:4:"0e5f";s:31:"static/simplePlaylist/setup.txt";s:4:"1f8b";}',
	'suggests' => array(
	),
);

?>